stable/aws-sigv4-proxy-admission-controller/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+name: sidecar-injector
```

Comment:
sidecar-injector is too generic. Lets use a consistent name here and in the documentation as well.